molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -180,6 +180,9 @@
 <div class="row">
 <div class="col-md-12">
 <div id="plugin-container">
+    <#if plugin_description??>
+        ${plugin_description}
```

Comment:
Why display the plugin description above every plugin?
- It will push down relevant content even more.
- I probably know what this plugin does since I selected it
- When I open another application (e.g. Excel) it doesn't start with a line telling me what Excel is.
- Displaying 'This is a plugin that displays static content stored in the database' above the Home plugin is not something I would want to see as a user.

I would expect to see the Plugin description in the menu manager plugin or when I select it as a data set.

Reply:
> It will push down relevant content even more.


It will only push down content if there is a description, and if there is no description, nothing is pushed down.

> I probably know what this plugin does since I selected it

Practice says this is not true, users tend to discover what the interface has to offer.

> When I open another application (e.g. Excel) it doesn't start with a line telling me what Excel is.

But expecting a user to look at the import wizard and then know how to create an EMX file etc is a bit 2 much. 

> Displaying 'This is a plugin that displays static content stored in the database' above the Home plugin is not something I would want to see as a user.

Adding a description for the home plugin seems to be pointless, so no one will ever do it? And if they do, well its their instance of MOLGENIS.

Everything about this description is 100% configurable. If you do not want it, then you don't have it.